examples/example_1/readme.md

```diff
@@ -0,0 +1,12 @@
+# Example 01
+## Use case
+Creation of payments/wire transfers from an internal data source using the Cardda platform
+## Behavior
+The script assumes the existance of environmental variables that define the origin bank_account and the bank_key used for creating the transactions.
+
+First the script collects the the information of each payment to be done from an internal SQL database. Then It takes care of verifying that each recipient for the payments already exists in Cardda, and has the proper status to begin doing transactions with it. If any particular recipient doesnt exists or has a status different from the ideal case, the script will try to create or update it properly (if possible).
```

Comment:
Hay un doble 'the' al comienzo :p
El 'It' después del primer punto tiene la 'i' en uppercase